com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md

```diff
@@ -12,7 +12,7 @@ The HDRP Camera shares many properties with the [Standard Unity Camera](https://
 
 | **Property**                    | **Description**                                              |
 | ------------------------------- | ------------------------------------------------------------ |
-| **Background Type**             | Use the drop-down to select the type of background that the Camera fills the screen with before it renders a frame.<br />&#8226; **Sky**: The Camera fills the screen with the sky defined in the [Visual Environment](Override-Visual-Environment.md) of the current [Volume](Volumes.md) settings.<br />&#8226; **Color**: The Camera fills the screen with the color set in **Background Color**.<br />&#8226; **None**: The Camera does not clear the screen and shows what it rendered in the previous frame as the background instead. |
+| **Background Type**             | Use the drop-down to select the type of background that the Camera fills the screen with before it renders a frame.<br />&#8226; **Sky**: The Camera fills the screen with the sky defined in the [Visual Environment](Override-Visual-Environment.md) of the current [Volume](Volumes.md) settings.<br />&#8226; **Color**: The Camera fills the screen with the color set in **Background Color**.<br />&#8226; **None**: The Camera does not clear the screen and the color buffer is left uninitialized. In this case, there are no guarantees on what the contents of the buffer will be (could be content from the previous frame, or content from another camera). This option should be used with caution. |
```

Comment:
<br />&#8226; **None**: The Camera does not clear the screen and the color buffer is left uninitialized. In this case, there are no guarantees on what the contents of the buffer are when you start drawing. It could be content from the previous frame or content from another camera. Because of this, use this option with caution.